shopinvader_product_variant_multi_link/models/shopinvader_variant.py

```diff
@@ -13,10 +13,10 @@ def _get_product_links(self):
         return self.product_variant_link_ids
 
     def _product_link_target(self, link):
-        if link.left_product_id != self.record_id:
-            return link.left_product_id
-        else:
+        if link.left_product_id == self.record_id:
             return link.right_product_id
+        elif link.type_id.is_symmetric:
```

Comment:
IMHO you should not test is_symmetric type.

The link type key is sufficent to filter which product to show.

@Cedric-Pigeon @simahawk

Reply:
@chafique-delli can you reply and rebase your PR ?